angular/src/services/jslib-services.module.ts

```diff
@@ -0,0 +1,416 @@
+            provide: LOCALE_ID,
+            useFactory: (i18nService: I18nServiceAbstraction) => i18nService.translationLocale,
+            deps: [I18nServiceAbstraction],
+        },
+        ValidationService,
+        AuthGuardService,
+        UnauthGuardService,
+        LockGuardService,
+        ModalService,
+        {
+            provide: AppIdServiceAbstraction,
+            useClass: AppIdService,
+            deps: [StorageServiceAbstraction],
+        },
+        {
+            provide: AuditServiceAbstraction,
+            useClass: AuditService,
+            deps: [CryptoFunctionServiceAbstraction, ApiServiceAbstraction],
+        },
+        {
+            provide: AuthServiceAbstraction,
+            useClass: AuthService,
+            deps: [
+                CryptoServiceAbstraction,
+                ApiServiceAbstraction,
+                UserServiceAbstraction,
+                TokenServiceAbstraction,
+                AppIdServiceAbstraction,
+                I18nServiceAbstraction,
+                PlatformUtilsServiceAbstraction,
+                MessagingServiceAbstraction,
+                VaultTimeoutServiceAbstraction,
+                LogService,
+                CryptoFunctionServiceAbstraction,
+                EnvironmentServiceAbstraction,
+                KeyConnectorServiceAbstraction,
+            ],
+        },
+        {
+            provide: CipherServiceAbstraction,
+            useFactory: (cryptoService: CryptoServiceAbstraction, userService: UserServiceAbstraction,
+                settingsService: SettingsServiceAbstraction, apiService: ApiServiceAbstraction,
+                fileUploadService: FileUploadServiceAbstraction, storageService: StorageServiceAbstraction,
+                i18nService: I18nServiceAbstraction, injector: Injector, logService: LogService) =>
+                new CipherService(cryptoService, userService, settingsService, apiService, fileUploadService,
+                    storageService, i18nService, () => injector.get(SearchServiceAbstraction), logService),
+            deps: [
+                CryptoServiceAbstraction,
+                UserServiceAbstraction,
+                SettingsServiceAbstraction,
+                ApiServiceAbstraction,
+                FileUploadServiceAbstraction,
+                StorageServiceAbstraction,
+                I18nServiceAbstraction,
+                Injector, // TODO: Get rid of this circular dependency!
+                LogService,
+            ],
+        },
+        {
+            provide: FolderServiceAbstraction,
+            useClass: FolderService,
+            deps: [
+                CryptoServiceAbstraction,
+                UserServiceAbstraction,
+                ApiServiceAbstraction,
+                StorageServiceAbstraction,
+                I18nServiceAbstraction,
+                CipherServiceAbstraction,
+            ],
+        },
+        { provide: LogService, useFactory: () => new ConsoleLogService(false) },
```

Comment:
A bit out of scope for this PR, but I've always thought it odd that this bool is hard coded. From memory it sets the log level based on whether you're running in development mode. Would be nice to detect this from the environment (if possible) so that it's more usable for debugging.

Reply:
I agree. We can't simply inject platformUtils though since that will cause a circular dependency. I think we'll need to extract the `isDev` logic since it's currently also the source of another circular dependency.

Reply:
`isDevService` 😆

Reply:
That's fair. But moving the `isDev` logic will also solve one of the circular dependencies. So probably worthwhile to still extract. But I think both those changes are out of scope for this PR.